tests/DocBlock/AnnotationTest.php

```diff
@@ -213,6 +213,46 @@ public function provideRemoveCases()
         return $cases;
     }
 
+    /**
+     * @param string $content
+     * @param string $expected
+     *
+     * @dataProvider provideRemoveEdgeCasesCases
+     */
+    public function testRemoveEdgeCases($content, $expected)
```

Comment:
please flip the arguments to `$expected, $input` to be consistent with the rest of the code base

Reply:
Done